src/basic/compress.c

```diff
@@ -155,11 +162,11 @@ int compress_blob_xz(const void *src, uint64_t src_size,
         assert(dst_size);
 
 #if HAVE_XZ
-        static const lzma_options_lzma opt = {
+        lzma_options_lzma opt = {
                 1u << 20u, NULL, 0, LZMA_LC_DEFAULT, LZMA_LP_DEFAULT,
                 LZMA_PB_DEFAULT, LZMA_MODE_FAST, 128, LZMA_MF_HC3, 4
         };
-        static const lzma_filter filters[] = {
+        const lzma_filter filters[] = {
                 { LZMA_FILTER_LZMA2, (lzma_options_lzma*) &opt },
```

Comment:
Please drop `const` attribute and the cast `(lzma_options_lzma*)`.
Then, please drop the cast in `sym_lzma_stream_buffer_encode()` below.

Reply:
Hey, please drop the cast.

Reply:
Not addressed. Please drop the cast. Or, is it necessary ?